src/metabase/db/custom_migrations.clj

```diff
@@ -1033,3 +1033,29 @@
 (define-reversible-migration UnifyTimeColumnsType
   (unify-time-column-type! :up)
   (unify-time-column-type! :down))
+
+(define-reversible-migration CardRevisionAddType
```

Comment:
we need to do this to make sure all the card revisions have the same schema as the active card.